applications/consdb/values-base.yaml

```diff
@@ -0,0 +1,13 @@
+environment:
+  KAFKA_REST: "https://base-lsp.lsst.codes/sasquatch-rest-proxy"
+  KAFKA_BOOTSTRAP: "sasquatch-base-kafka-bootstrap.lsst.codes"
+  vaultUrl: "https://vault.lsst.codes"
```

Comment:
I don't see any `environment` key in `values.yaml`. Is this old config? I'd suggest removing it if not used.

Reply:
Seems like the Kafka settings are going in via the root `kafka` key?

Reply:
yes my mistake thanks for catching  environment --> kafka